test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -317,3 +317,480 @@ test.describe( 'Navigation block', () => {
 		} );
 	} );
 } );
+
+test.describe( 'List view editing', () => {
+	test( 'it should show a list view in the inspector controls', async ( {
```

Comment:
```suggestion
	test( 'shows list view in the inspector controls', async ( {
```

In most frameworks `test` is just an alias for `it`, so the word `it` doesn't need to be in the description itself. I always think removing the word 'should' makes the description shorter and easier to read, so also suggesting that.

Maybe I should add some of this to the docs.

Reply:
✅